docs/docs/cli-reference/service-restart.md

```diff
@@ -0,0 +1,13 @@
+---
+title: service restart
+sidebar_label: service restart
+slug: /service-restart
+---
+
+Services can be restarted like so:
+
+```bash
+kurtosis service restart $THE_ENCLAVE_IDENTIFIER $THE_SERVICE_IDENTIFIER
+```
+
+where `$THE_ENCLAVE_IDENTIFIER` and the `$THE_SERVICE_IDENTIFIER` are [resource identifiers](../advanced-concepts/resource-identifier.md) for the enclave and service, respectively.
```

Comment:
could we mention that a user can pass more than one service identifier?

Reply:
I will do that!